pkg/inmemorychannel/message_dispatcher_benchmark_test.go

```diff
@@ -76,12 +76,6 @@ func BenchmarkDispatcher_dispatch_ok_through_2_channels(b *testing.B) {
 		},
 	}
 
-	// Let's mock this stuff!
-	httpSender, err := kncloudevents.NewHTTPMessageSenderWithTarget(channelA.URL.String())
-	if err != nil {
-		b.Fatal(err)
-	}
-
```

Comment:
We have to switch to the new message sender here. Check the following for an example: 

https://github.com/knative/eventing/blob/90721ab4054f4ec239f84371e8afdb1b6fcdfc5a/pkg/broker/filter/filter_handler.go#L299-L303

Reply:
@creydr  Updated, Please verify in the review.

Thank you.